cdserver/app/models/address.rb

```diff
@@ -1,3 +1,4 @@
+# street address
 class Address < ApplicationRecord
-  belongs_to :shelter_building
```

Comment:
Many things can have an address.  A person can have an address.  A shelter building.  A house.  I don't think we can tie an address to any one thing.